src/snackbar.tsx

```diff
@@ -42,7 +42,7 @@ export type ImperativeHandle = {
 const SnackbarComponent = React.forwardRef<ImperativeHandle, Props>(
     ({message, buttonText, duration, onClose, type, withDismiss = false, dataAttributes}, ref) => {
         const {texts} = useTheme();
-        const [isOpen, setIsOpen] = React.useState(false);
+        const [isOpen, setIsOpen] = React.useState(process.env.NODE_ENV === 'test');
```

Comment:
This should fix https://github.com/Telefonica/global-checkout-webview/blob/371d2958beb0c61b69c804b441038ace7061bb2a/web/app/invoice-payment/components/__tests__/invoice-selection.test.tsx#L218

Reply:
are you sure this is needed? we don't have issues in our snackbar related unit tests